docs/docsite/constraints.in

```diff
@@ -0,0 +1,4 @@
+sphinx == 5.1.1
+sphinx-ansible-theme == 0.9.1
+docutils == 0.16
+Jinja2 < 3.1
```

Comment:
I don't think you need these. I introduced this concept of an additional constraint file specifically for `ansible/ansible-documentation` because they want to be extra strict about their updates and have a manual testing process for updating most of these entries or have known incompatibilities that won't be fixed anytime soon.

It seems to be that awx doesn't have the same restrictions so I'd just remove this file.

The only entry that caught my eye is Jinja2 which has an upper version boundary. If it is really necessary, then I'd keep this file. But please add a comment to each individual entry here, explaining why the thing is listed and perhaps, when it can be removed.

Reply:
@webknjaz Thanks for the review here!

I would really like to get rid of the constraints file but I wasn't sure if we could remove the boundaries yet, especially Jinja2. It's somewhat hard to explain exactly why it exists because it has been around longer than I've been with the project. It mostly relates to that swagger module.

I would like to get to the point where there is no need for pins. Maybe we should just drop the constraints.in in this PR and shoot for the removal of the pins but I think it's going to take a bit more testing and I would rather get everything integrated first, like the swagger bits, then remove the constraints file. Does that sound OK?

Also I did start exploring why each entry is listed but, again, a lot of this predates my involvement so I'm guessing a little.

Reply:
I've dropped the extra constraints.in file.